### DISCUSSION TITLE
Large Numbers of Databases

### DISCUSSION THREAD

**Discussion by Eric24** [ArcadeData/arcadedb#1359] [General]
What are the practical limits to the number of databases on one ArcadeDB server or cluster? What are the server-side implications (i.e. memory overhead per active database, etc.)? What are the client-side implecations (and specifically if separate connections are necessary per database, or if a connection can access different databases)? (And maybe this doesn't apply to the REST interface).

**Comment by lvca** [comment]
Hi @Eric24, it all depends on the database size, usage and hw/sw configuration where ArcadeDB Server is installed. I suggest to do some tests. As a reference, we have some users in production with 200K databases per server, where not all the databases are accessed at the same time. We have a pending enhancement to implement a better allocation of unused databases (using a LRU + timeout) for the future.
